Vagrantfile

```diff
@@ -125,9 +125,16 @@ end
 
 # Load the config file before the second section of the splash screen
 
-if File.file?(File.join(vagrant_dir, 'vvv-custom.yml')) == false then
-  puts "#{yellow}Copying #{red}vvv-config.yml#{yellow} to #{green}vvv-custom.yml#{yellow}\nIMPORTANT NOTE: Make all modifications to #{green}vvv-custom.yml#{yellow} in future so that they are not lost when VVV updates.#{creset}\n\n"
-  FileUtils.cp( File.join(vagrant_dir, 'vvv-config.yml'), File.join(vagrant_dir, 'vvv-custom.yml') )
+vvv_config_file = File.join(vagrant_dir, 'config/config.yml')
```

Comment:
We also need to copy `vvv-custom.yml` to `config/config.yml` if it exists for backwards compatibility

Reply:
uh I missed that

Reply:
But it is copying `vvv-custom.yml` to `config/config.yml` for backwards compatibilty.

What am I missing?

Reply:
hmm I've misunderstood, it needs to copy `config/default-config.yml` to `config/config.yml` too


```
if config/config.yml doesn't exist
    if vvv-custom.yml exists
        copy that to config/config.yml
    else
        copy config/default-config.yml to config/config.yml
```

It should also mention the filenames explicitly so we don't confuse any users

Reply:
This is what's it's currently doing with my changes

```
if vvv-custom.yml exists
   copy that to config/config.yml

if config/config.yml doesn't exist
    copy config/default-config.yml to config/config.yml
```

Pretty much the same.

Reply:
I've committed an update that refactors that bit

Reply:
Awesome, thanks!